app/controllers/key_pairs_controller.rb

```diff
@@ -60,11 +92,15 @@ def update
 
     respond_to do |format|
       if @key_pair.update_attributes(key_pair_params)
-        format.html { redirect_to @key_pair, notice: 'Key pair was successfully updated.' }
+        format.html do redirect_to @key_pair,
+                                   notice: 'Key pair was successfully updated.'
+                    end
```

Comment:
`end` at 97, 20 is not aligned with `format.html do redirect_to @key_pair,` at 95, 8